mod/common.php

```diff
@@ -45,37 +46,36 @@ function common_content(App $a) {
 			intval($uid)
 		);
 		/// @TODO Handle $c with DBM::is_result()
-
-		$vcard_widget .= replace_macros(get_markup_template("vcard-widget.tpl"),array(
+		$vcard_widget = replace_macros(get_markup_template("vcard-widget.tpl"), array(
 			'$name' => htmlentities($c[0]['name']),
 			'$photo' => $c[0]['photo'],
 			'url' => 'contacts/' . $cid
 		));
 
-		if (! x($a->page,'aside')) {
+		if (!x($a->page, 'aside')) {
 			$a->page['aside'] = '';
 		}
 		$a->page['aside'] .= $vcard_widget;
 	}
 
-	if (! DBM::is_result($c)) {
+	if (!DBM::is_result($c)) {
 		return;
 	}
 
-	if(! $cid) {
-		if(get_my_url()) {
-			$r = q("SELECT `id` FROM `contact` WHERE `nurl` = '%s' AND `uid` = %d LIMIT 1",
-				dbesc(normalise_link(get_my_url())),
-				intval($profile_uid)
+	if (!$cid && get_my_url()) {
+		/// @todo : Initialize $profile_uid
+		$r = q("SELECT `id` FROM `contact` WHERE `nurl` = '%s' AND `uid` = %d LIMIT 1",
+			dbesc(normalise_link(get_my_url())),
+			intval($profile_uid)
```

Comment:
@annando Any idea how to properly initialize `$profile_uid` here?

Reply:
Isn't this the local_user() in most cases?

Reply:
Maybe, but this is the module showing the common friends between two users/contacts, so there are many possibilities.

Reply:
Common friends is done differently. But for a deeper look I'm too tired now.

Reply:
That's okay, it's been broken like this for a while, it can wait a little longer.